### ISSUE TITLE
Old version of Npcap on winget

### ISSUE THREAD

**Issue by StepanKuklis** [nmap/npcap#732]
Hello, 
there is 0.86 (from 2016) version of npcap on winget package manager. 
```
λ  winget search npcap
Name  Id             Version Source
Npcap Insecure.Npcap 0.86    winget
λ  winget show Insecure.Npcap
Found Npcap [Insecure.Npcap]
Version: 0.86
Publisher: Insecure
Publisher Url: https://insecure.org/fyodor/
Description: Nmap Project's Windows packet capture and transmission library
Homepage: https://nmap.org/npcap/
License: Proprietary
Privacy Url: https://insecure.org/privacy.html
Copyright: Copyright 2016 Insecure.Com LLC ("The Nmap Project")
Installer:
  Installer Type: nullsoft
  Installer Url: https://nmap.org/npcap/dist/npcap-0.86.exe
  Installer SHA256: ab677b9037dc377303e6d68a9b8a8195c8e976efd550a6bb6ce91eed551c5cbe
```
Im not sure if you know. If you do, I think it is better to have no version there than a very old version.

I found an older discussion here about distributing the OEM version, but this is unrelated (I think) so Im opening this new one.
https://github.com/nmap/npcap/issues/588

I found this in winget issues
https://github.com/microsoft/winget-pkgs/issues/6494

**Comment by fyodor** [comment]
Yeah we absolutely don't want Npcap (especially Version 0.86) on Winget.  That's more than 7 years old and is subject to dozens of critical bugs that we've fixed since then (see [the Npcap Changelog](https://npcap.com/changelog) for details.  The cleanest approach would be to convince Winget folks to remove it.  But we may also have to look at technical countermeasures such as putting the old versions of Npcap behind a captcha and/or maybe even adding one to the installer.  I'd hate to just remove the old versions from our download directory since they do have occasional valid uses.

**Comment by cachius** [comment]
I just saw that @kannkyo added v1.81 to [winget-pkgs/Insecure/Npcap](/microsoft/winget-pkgs/tree/master/manifests/i/Insecure/Npcap) two days ago on April 15, 2025.  
By default v1.81 is installed, v0.86 is still available behind `--version 0.86`. 

A reasoned PR against [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs) could remove v0.86 from there.

**Comment by nullifyac** [comment]
version is still 0.86

**Comment by fyodor** [comment]
Thanks everybody.  We also don't want Winget, Chocolatey, or similar systems installing ancient versions of Npcap.  In fact nobody should really be using those versions anymore, so I just moved them out of the npcap.com dist directory.  If someone needs an ancient copy of Npcap for some reason, they can reach out to me directly.

I just tested with Winget and it now gives an error, which I think is much better than installing Npcap 0.96:

> PS C:\Users\fyodor> winget install npcap
> Found Npcap [Insecure.Npcap] Version 0.96
> This application is licensed to you by its owner.
> Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
> Downloading https://npcap.com/dist/npcap-0.96.exe
> An unexpected error occurred while executing the command:
> Download request status is not success.
> 0x80190194 : Not found (404).

Regarding that terrible "Insecure" namespace, I guess that's on us for originally being named "Insecure.Com LLC".   We did change our name several years ago to "Nmap Software LLC" which hopefully sounds more trustworthy!

I am closing this as resolved.  Cheers!

**Comment by vaguue** [comment]
> Thanks everybody. We also don't want Winget, Chocolatey, or similar systems installing ancient versions of Npcap. In fact nobody should really be using those versions anymore, so I just moved them out of the npcap.com dist directory. If someone needs an ancient copy of Npcap for some reason, they can reach out to me directly.
> 
> I just tested with Winget and it now gives an error, which I think is much better than installing Npcap 0.96:
> 
> > PS C:\Users\fyodor> winget install npcap
> > Found Npcap [Insecure.Npcap] Version 0.96
> > This application is licensed to you by its owner.
> > Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
> > Downloading https://npcap.com/dist/npcap-0.96.exe
> > An unexpected error occurred while executing the command:
> > Download request status is not success.
> > 0x80190194 : Not found (404).
> 
> Regarding that terrible "Insecure" namespace, I guess that's on us for originally being named "Insecure.Com LLC". We did change our name several years ago to "Nmap Software LLC" which hopefully sounds more trustworthy!
> 
> I am closing this as resolved. Cheers!

I was using npcap 0.86 for my ci/cd pipeline for my open-source project https://github.com/vaguue/over-the-wire. Can I get it please?

**Comment by cachius** [comment]
@vaguue You can get it from archive.org searching for the URL from OP. Old winget manifest is at https://github.com/tratintueqte/winget-pkgs/blob/e87895fd9ce08301ff29cf6bca5b181ce6dcb13b/manifests/i/Insecure/Npcap/0.86/Insecure.Npcap.installer.yaml if you want to build a local manifest.
